apps/tangle-dapp/containers/NominatorStatsContainer/NominatorStatsContainer.tsx

```diff
@@ -137,13 +137,13 @@ const NominatorStatsContainer: FC = () => {
               ) : (
                 <>
                   <Link href={WEBB_TANGLE_DOCS_STAKING_URL} target="_blank">
-                    <Button variant="utility" className="w-full">
+                    <Button variant="utility" className="!min-w-[150px]">
```

Comment:
Question: What's causing the need to force the `min-w` classes? Maybe we need to change `Button` component to always prioritized the classed that was passed in? 🤔